src/ui/utils.rs

```diff
@@ -39,7 +39,7 @@ pub fn style_default(light: bool) -> Style {
   }
 }
 pub fn style_logo() -> Style {
-  Style::default().fg(Color::Blue)
+  Style::default().fg(Color::Green)
```

Comment:
I can revert whatever you don't like.

Reply:
These are personal preferences so instead of this I would rather inplement #68 (The idea is to use just a few colors everywhere and then make it customizable via a toml file like `~/.kdash/config.toml`)

Reply:
Yes and no. The color itself is chosen by me, that's correct.
But I changed the places that have low contrast differences.

3.5:1 or 4:1 instead of 1.5:1 (the old color is fine only on white)

![Screenshot from 2022-01-24 16-40-23](https://user-images.githubusercontent.com/45714/150814880-b17777b6-159b-4a9f-83b6-2a6309e576b7.png)
![Screenshot from 2022-01-24 16-40-11](https://user-images.githubusercontent.com/45714/150814908-092eeb9d-4a9d-40c4-ad19-1fa0aabbc520.png)
![Screenshot from 2022-01-24 16-39-57](https://user-images.githubusercontent.com/45714/150814912-bf70e0e3-ff9a-4858-a398-dd3ac4de1821.png)

Reply:
See the same rendered in Yakyake vs Konsole with same theme settings

![image](https://user-images.githubusercontent.com/1107223/150817274-966b1678-41fa-4c92-8fd3-269feb5852d9.png)
![image](https://user-images.githubusercontent.com/1107223/150817301-9b2e32b9-b69f-43c4-9037-9ea1ae81a805.png)

Reply:
Yes, it's because it's `GREEN`, it will not be like this if we choose `rgb(0, 255, 0)`
The same that the BG color is the same in both pictures above.

Reply:
oh shit, I never realized that :facepalm:

Reply:
Anyway, I like github put the green dot next to the color, test: `rgb(34,67,233)` and a wrong one: `rgb(234,5,345)`

Reply:
Didn't know that GitHub does that cool